app/controllers/admin/applications_controller.rb

```diff
@@ -1,13 +1,42 @@
 class Admin::ApplicationsController < ApplicationController
   def show
     @application = Application.find(params[:id])
-    @application_pets = @application.pets
-    @app_petapp = @application.pet_applications
+    @pet_applications = PetApplication.find_applications(@application.id)
   end
 
   def update
-    application = PetApplication.find(params[:application_id])
-    application.update(pet_applications_status: params[:pet_applications_status])
-    redirect_to "/admin/applications/#{application.application_id}"
+    if params[:updated_status] == "Approved"
+      pet_application = PetApplication.find(params[:pet_application_id])
+      # how to better utilize private params here
+      pet_application.update(pet_applications_status: params[:updated_status])
+      redirect_to "/admin/applications/#{pet_application.application_id}"
+    elsif params[:updated_status] == "Rejected"
+      pet_application = PetApplication.find(params[:pet_application_id])
+      pet_application.update(pet_applications_status: params[:updated_status])
+      redirect_to "/admin/applications/#{pet_application.application_id}"
+    end
   end
+
+  ## How and why would you utilize private params here and/or our find_applications class method from PetApplication
```

Comment:
Maybe we will get private params to function better in the future but good work troubleshooting this idea - everything is working now!